lib/config/configuration.js

```diff
@@ -745,11 +742,7 @@ Configuration.prototype._loadExcludedFiles = function(masks) {
  * @protected
  */
 Configuration.prototype._loadExtract = function(masks) {
-    if (masks === false) {
```

Comment:
I think should be possible to pass `true` value to use default mask for html files.